src/scala/com/github/johnynek/bazel_deps/Resolver.scala

```diff
@@ -77,7 +82,10 @@ class Resolver(servers: List[MavenServer], resolverCachePath: Path) {
      * and do the sha1.
      */
     def toArtifactRequest(m: MavenCoordinate, extension: String): ArtifactRequest = {
-      val classifier = null // We don't use this
+      val classifier = m.classifier match {
```

Comment:
can we remove the duplication here and instead add:

```scala
def toNonEmptyString: Option[String] =
  if (asString.isEmpty) None else Some(asString)
```

Then you could use `m.classifier.toNonEmptyString.orNull`.

Is that alright rather than repeating this match twice?